Problems/Set-1/6/main.cpp

```diff
@@ -0,0 +1,41 @@
+
+
+
+#include <iostream>
+#include <sstream>
+
+using namespace std;
+
+/***
+ * 
+ */ 
+
+void line(bool state,int n){
+	stringstream line;
+	for(int i=0;i<n;i++){
+		line<<((state)?".":"#");
+		line<<((state)?".":"#");
+		state=!state;
+	}
+	string li = line.str();
+	cout<<li<<endl<<li<<endl;
```

Comment:
Avoid (why?) using endl inside a function that's called (non-constant)  "t" times from another function. Use "\n" instead.